### PR TITLE
New syringe sprites

### PR DESCRIPTION
**Pull request by MTandi**
dreamseeker_VLjPCpzfo7

## About The Pull Request

Resprites syringes.

## Why It's Good For The Game

Syringes were too small and hard to click, now they're easier to click and the reagent color can be seen better.

## Changelog

:cl:
image: New sprite for syringes
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
I really like these but like I said in the discord i feel the bluespace syringe one specifically feels.. lacking. Compare it to the bluespace beaker, they're extremely different-looking in a way that IMO leaves the resprite looking worse, this one almost feels like it's made OF bluespace crystals rather than machinery fueled by it.

You could use this opportunity to give large syringes like the lethal injection one an unique sprite also if you want

**Comment by murder357** [comment]
The lore as to why Nanotransen had to make bigger syringes:

"Because of the increased robustness in Space Station 13, cross-species evolution has caused everyone veins to be bigger, so we hope you enjoy these brand new, upgraded syringes!"

super good lore aside, I don't think this was required but I don't mind it either.

**Comment by Pumpkinoe** [comment]
> I really like these but like I said in the discord i feel the bluespace syringe one specifically feels.. lacking.

to be fair there's not much to work with on a syringe sprite, you have two whole sprites for the handle

**Comment by Livrah** [comment]
Can this be shot with Nukie's 84mm Rocket Launcher?

**Comment by MTandi** [comment]
> Can this be shot with Nukie's 84mm Rocket Launcher?

No changes in code, except that crude syringe doesn't show reagent color anymore.

**Comment by cnleth** [comment]
I don't think the sprites being bigger is better, they are now comparable in size to shotgun shells and look odd because of that.

**Comment by MTandi** [comment]
> I don't think the sprites being bigger is better, they are now comparable in size to shotgun shells and look odd because of that.

They're also comparable in size with medipens and many other items, what's odd about this?

**Comment by cnleth** [comment]
> > I don't think the sprites being bigger is better, they are now comparable in size to shotgun shells and look odd because of that.
> 
> They're also comparable in size with medipens and many other items, what's odd about this?

Syringes probably should be narrow and long, and these are wide, so I like the old ones more, but that's just my opinion. The sprites are not bad, something just feels wrong to me

**Comment by CoiledLamb** [comment]
i'm still not a fan of these, i don't think the additional pixel of the height adds to the design, rather it detracts from the common visual language of syringes. as stated in tg discord, a better way to make them easier to click would be adding 1-2 transparent pixels to each side, so that players can click it easier, without making the sprite look worse.

**Comment by TheVekter** [comment]
> I don't think the sprites being bigger is better, they are now comparable in size to shotgun shells and look odd because of that.

Easier to click on. Old syringes were kind of annoying. This is good.



> i'm still not a fan of these, i don't think the additional pixel of the height adds to the design, rather it detracts from the common visual language of syringes. as stated in tg discord, a better way to make them easier to click would be adding 1-2 transparent pixels to each side, so that players can click it easier, without making the sprite look worse.

Can you be more specific on your criticism? It doesn't really help much if all you have to say is that you don't think it looks like a syringe.

**Comment by CoiledLamb** [comment]
> > i'm still not a fan of these, i don't think the additional pixel of the height adds to the design, rather it detracts from the common visual language of syringes. as stated in tg discord, a better way to make them easier to click would be adding 1-2 transparent pixels to each side, so that players can click it easier, without making the sprite look worse.
> 
> Can you be more specific on your criticism? It doesn't really help much if all you have to say is that you don't think it looks like a syringe.
okay,  here's my specific criticism: making the syringes 1 pixel taller does not improve the sprite, because most people associate syringes with symmetrical objects with a needle poking out the center. i don't think the shading adds a lot here, that the base syringe wasn't already doing. and i think the scale actively detracts from the design. 
what would be good is just adding transparent pixels on to the syringe sprite, so that OP can make the object as easy as they want it to be to click on, without touching the actual look of the sprite itself. one pixel of height is not going to change anything with how clickable they are, in my book.

**Comment by Cheshify** [comment]
These look more like syringes and more readable. Good PR

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -145,7 +145,7 @@
 
 /obj/item/reagent_containers/syringe/update_overlays()
 	. = ..()
-	if(reagents?.total_volume)
+	if(reagents?.total_volume && !istype(src, /obj/item/reagent_containers/syringe/crude))
```
Comment (Mothblocks) [inline code/modules/reagents/reagent_containers/syringes.dm:148]: You should instead move out the body of this proc (everything except `..()`) into its own proc that `/crude` then overrides to be a no-op.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/syringes.dm (hunk 1 above)
Comment (MTandi) [reply code/modules/reagents/reagent_containers/syringes.dm]: Please confirm that I did what you asked.

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 2]
```diff
@@ -145,6 +145,9 @@
 
 /obj/item/reagent_containers/syringe/update_overlays()
 	. = ..()
+	. += update_reagent_overlay()
+
+/obj/item/reagent_containers/syringe/proc/update_reagent_overlay()
```
Comment (Mothblocks) [inline code/modules/reagents/reagent_containers/syringes.dm:150]: Auto-doc

**Review by Mothblocks — APPROVED** [review]

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 3]
```diff
@@ -147,6 +147,7 @@
 	. = ..()
 	. += update_reagent_overlay()
 
+///Used to determine whether syringe can show reagent overlay
```
Comment (Mothblocks) [inline code/modules/reagents/reagent_containers/syringes.dm:150]: This documentation isn't correct

```suggestion
/// Returns a list of overlays to add that relate to the reagents inside the syringe
```

**Pull request merged**